### PR TITLE
create crosswalks; save as internal data

### PR DESCRIPTION
**Pull request by erikarasnick**

### CODE REVIEW

File: data-raw/crosswalks/neighborhood_to_neighborhood.R [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+neigh_sna
+neigh_cchmc
+
+neighs <- mapview::mapview(neigh_ccc, col.regions = "red", legend = FALSE) +
+  mapview::mapview(neigh_sna, col.regions = "blue", legend = FALSE) +
+  mapview::mapview(neigh_cchmc, col.regions = "green", legend = FALSE)
+
+# do we expect users to convert neighborhood data to other neighborhoods? These are all
+# different, but usually only slightly... would end up with very small weights in a
+# lot of cases
```
Comment (cole-brokamp) [inline data-raw/crosswalks/neighborhood_to_neighborhood.R]: was this left over from our discussion about neighborhood to neighborhood mappings?   we can delete this, right?

Comment (cole-brokamp) [reply data-raw/crosswalks/neighborhood_to_neighborhood.R]: also can remove the `.DS_Store` files (or add to `.gitignore`)

**Review by cole-brokamp — CHANGES_REQUESTED** [review]
How are we envisioning that we will access these from within the CODEC tools?  Will they all be globally available once we `library(cincy)`?

Also, I think we should extend this pull request to include an example usage in `README.Rmd`.  This would help us think about how to best make them available within R code.

**Comment by erikarasnick** [comment]
The way it is currently written, we would have to use the triple colon to access the crosswalk datasets. I think we had discussed keeping them internal data here, then using them in exported functions in CODECtools. I'm starting to think the crosswalks and the crosswalk functions should be in the same package... what do you think? Then the example of how to use the crosswalks would go in whichever package that is. It seems weird to me to keep the crosswalks here, write the crosswalk functions in CODECtools, and show their example use here.

**Comment by cole-brokamp** [comment]
When I install from this branch, I can only see the following in the `:::` namespace:

image

and I see this when trying to access them:

```r
r$> cincy::tract_to_zcta_2020
Error: 'tract_to_zcta_2020' is not an exported object from 'namespace:cincy'

r$> cincy:::tract_to_zcta_2020
Error: object 'tract_to_zcta_2020' not found

```

**Comment by cole-brokamp** [comment]
But, I agree on making the functions and weights in the same package.  Let's keep working on this through this PR and then talk more tomorrow about where it should live long-term.

**Comment by erikarasnick** [comment]
I know I ran the code for all the crosswalks, but somehow they didn't actually get stored in `sysdata.rda`. Now they should be accessible with the triple colon.

**Review by cole-brokamp — APPROVED** [review]

**Pull request merged**